docs/Guide.Migration.md

```diff
@@ -5,6 +5,20 @@ title: Migration Guide
 
 We are improving detox API as we go along, sometimes these changes require us to break the API in order for it to make more sense. These migration guides refer to breaking changes.
 
+## Migrating to 12.7.0 from older (nonbreaking)
+
+**This is only relevant to those running Detox using [`Jest` as the test runner](APIRef.Configuration.md#test-runner-configuration)**!
+
+In `12.7.0` we've greatly improved our support in Jest - trying to tackle these two caveats which hold developers back from embracing it:
```

Comment:
Sounds more like a CHANGELOG, not a migration guide. Maybe you should more focus on what to edit in the e2e/config.json?